apps/condo/domains/banking/queries/BankSyncTask.graphql

```diff
@@ -0,0 +1,69 @@
+query getBankSyncTasks ($where: BankSyncTaskWhereInput!) {
```

Comment:
Why do you need such queries? Seems like you have common fetched below....

And why raw where is passed? Is it possible to be more specific?